store/postgres/tests/store.rs

```diff
@@ -1533,6 +1540,24 @@ fn entity_changes_are_fired_and_forwarded_to_subscriptions() {
             )
             .unwrap();
 
+        // FIXME: If we produce fewer than 2 block ticks, the test hangs
```

Comment:
@timmclean @leodasvacas Any clever ideas here?

Reply:
I don't think failure by timeout is so bad, we can put a fixed [timeout](https://docs.rs/tokio/0.1.14/tokio/prelude/trait.StreamExt.html#method.timeout) around it.